### PR TITLE
RESOURCE-114 Support azure service fabric mesh volumes

### LINKED ISSUE #574: Service Fabric Mesh Volumes
https://docs.microsoft.com/en-us/rest/api/servicefabric/sfmeshrp-index-volumes

### PR DESCRIPTION
**Pull request by sathish-progress**
### Description

Support azure service fabric mesh volume(s)

### Issues Resolved
#574 

### Check List

- [X] New functionality includes tests
- [X] All tests pass
- [X] `rake lint` passes
- [X] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

**Review by soumyo13 — CHANGES_REQUESTED** [review]
Terraform files are missing.

**Review by soumyo13 — APPROVED** [review]

File: test/integration/verify/controls/azure_service_fabric_mesh_volume.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+rg = input(:resource_group, value: '')
+location = input(:location, value: '')
+
+control 'test the properties of all Azure Service Fabric Mesh Applications' do
```
Comment (soumyo13) [inline test/integration/verify/controls/azure_service_fabric_mesh_volume.rb:4]:
```suggestion
skip_control 'test the properties of all Azure Service Fabric Mesh Applications' do
```

File: test/integration/verify/controls/azure_service_fabric_mesh_volumes.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+location = input(:location, value: '')
+
+control 'test the properties of all Azure Service Fabric Mesh Applications' do
```
Comment (soumyo13) [inline test/integration/verify/controls/azure_service_fabric_mesh_volumes.rb:3]:
```suggestion
skip_control 'test the properties of all Azure Service Fabric Mesh Applications' do
```

File: test/integration/verify/controls/azure_service_fabric_mesh_volume.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+rg = input(:resource_group, value: '')
+location = input(:location, value: '')
+
+skip_control 'test the properties of all Azure Service Fabric Mesh Applications' do
```
Comment (soumyo13) [inline test/integration/verify/controls/azure_service_fabric_mesh_volume.rb:4]:
```suggestion
skip_control 'Test the properties of the Azure Service Fabric Mesh Application' do
```

Comment (soumyo13) [inline test/integration/verify/controls/azure_service_fabric_mesh_volume.rb]:
```suggestion
skip_control 'test the properties of all Azure Service Fabric Mesh Volumes' do
```

File: test/integration/verify/controls/azure_service_fabric_mesh_volume.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+rg = input(:resource_group, value: '')
+location = input(:location, value: '')
+
+skip_control 'Test the properties of the Azure Service Fabric Mesh Application' do
```
Comment (soumyo13) [inline test/integration/verify/controls/azure_service_fabric_mesh_volume.rb:4]:
```suggestion
skip_control 'Test the properties of the Azure Service Fabric Mesh Volume' do
```

File: test/integration/verify/controls/azure_service_fabric_mesh_volumes.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+location = input(:location, value: '')
+
+skip_control 'test the properties of all Azure Service Fabric Mesh Applications' do
```
Comment (soumyo13) [inline test/integration/verify/controls/azure_service_fabric_mesh_volumes.rb:3]:
```suggestion
skip_control 'test the properties of all Azure Service Fabric Mesh Volumes' do
```

**Review by IanMadd — APPROVED** [review]

**Pull request merged**